pkg/sentry/socket/netfilter/netfilter.go

```diff
@@ -50,7 +50,7 @@ type metadata struct {
 
 // nflog logs messages related to the writing and reading of iptables.
 func nflog(format string, args ...interface{}) {
-	log.Infof("netfilter: "+format, args...)
+	log.Debugf("netfilter: "+format, args...)
```

Comment:
I recommend guarding this behind if log.IsLogging(log.Debug). Otherwise you’re going to generate a lot of garbage, since the string concatenation is going to build a new object every time.